src/test/testdata/lsl/charLiterals.lsl

```diff
@@ -0,0 +1,13 @@
+libsl "1.0.0";
+library simple;
+typealias Int = int32;
+typealias Char = char;
+
+automaton A : Int {
+    var ch1: Char = '\u0041';
+    var ch2: Char = 'a';
+    var ch3: Char = 52;
```

Comment:
I don't think it is necessary here.
This line doesn't look like the value should be parsed as `char` and not as `int32`.

Reply:
Was deleted